.github/CHANGELOG.md

```diff
@@ -126,6 +126,9 @@
 1. [Sound] Improved nose wheel liftoff sound - @hotshotp (Boris)
 1. [Sound] Added new APU start contactor sounds - @hotshotp (Boris)
 1. [Sound] Added new realistic LEAP-1A engine sounds - @hotshotp (Boris)
+1. [Lights] Interior Improvements->  Back Lighting - @Snapmatics (Harry)
+1. [Lights] New Features->  Flood Lights/Reading Lights - @Snapmatics (Harry)
+1. [Lights] Exterior Improvements ->  Take-Off Lights/Landing Lights/Taxi Lights/Runway Lights/Strobe Lights/Beacon Lights/Navigation Lights - @Snapmatics (Harry)
```

Comment:
Just use simple sentences for changelogs that start with action verbs (fix, add, improve)